draft-ietf-rats-corim.md

```diff
@@ -1080,84 +1080,112 @@ The verification procedure is divided into three separate phases:
 * Evidence collection
 * Evidence appraisal
 
-At a few well-defined points in the procedure, the Verifier behaviour will depend on the specific CoRIM profile.
-Each CoRIM profile MUST provide a description of the expected Verifier behavior for each of those well-defined points.
+At a few well-defined points in the procedure, the Verifier behaviour will
+depend on the specific CoRIM profile.
+Each CoRIM profile MUST provide a description of the expected Verifier behavior
+for each of those well-defined points.
 
 Note that what follows describes a simplified implementation.
-Verifiers claiming compliance with this specification must exhibit the same externally visible behavior as described here,
+Verifiers claiming compliance with this specification must exhibit the same
+externally visible behavior as described here,
 they are not required to use the same internal data structures.
-For example, it is expected that the resources used during the initialisation phase can be amortised across multiple appraisals.
+For example, it is expected that the resources used during the initialisation
+phase can be amortised across multiple appraisals.
 
 ## Appraisal Context initialisation
 
-The goal of the initialisation phase is to load the CoRIM Appraisal Context with objects such as CoRIM tags,
-cryptographic validation key material (e.g., raw public keys, root certificates, intermediate CA certificate chains), etc.
-that will be used in the subsequent Evidence Appraisal phase.
+The goal of the initialisation phase is to load the CoRIM Appraisal Context
+with objects such as CoRIM tags,
+cryptographic validation key material (e.g., raw public keys, root certificates,
+intermediate CA certificate chains), etc. that will be used in the subsequent
+Evidence Appraisal phase.
```

Comment:
An observation: the term "CoRIM tag" does not look right.  A CoRIM _has_ tags (i.e., CoMIDs, CoSWIDs, CoBOMs, CoTS), but it's not itself a tag.

In this context seems more appropriate to talk about endorsed values and reference values rather than a generic CoRIM, which includes the crypto material.

```suggestion
with objects such as CoRIM tags,
cryptographic validation key material (e.g., raw public keys, root certificates,
intermediate CA certificate chains), etc. that will be used in the subsequent
Evidence Appraisal phase.
```